src/brag/client/ceph-brag

```diff
@@ -258,7 +258,7 @@ def get_nums():
   num_mons = len(oj['monmap']['mons'])
   num_osds = int(oj['osdmap']['osdmap']['num_in_osds'])
   try:
-    num_mdss = oj['mdsmap']['in']
+    num_mdss = oj['fsmap']['in']
```

Comment:
I'm not sure if this tool is supposed to be backwards compatible with older clusters -- if it is, then this code should check for both mdsmap and fsmap. the mdsmap->fsmap change happened in jewel.